### PR TITLE
Renaming docs to documents and cluster_alis to alias

### PR DESCRIPTION
**Pull request by charyeezy**

### CODE REVIEW

File: relevanceai/vector_tools/cluster.py [hunk 1]
```diff
@@ -683,7 +686,7 @@ def hdbscan_cluster(
                 ".".join([cluster_field, vector_fields[0], alias])
             )
         # load the documents
-        docs = self._get_all_documents(
+        documents = self.get_all_documents(
```
Comment (boba-and-beer) [inline relevanceai/vector_tools/cluster.py:689]: It's strange that this changed was reverted.

File: relevanceai/vector_tools/cluster.py [hunk 2]
```diff
@@ -686,7 +686,11 @@ def hdbscan_cluster(
                 ".".join([cluster_field, vector_fields[0], alias])
             )
         # load the documents
+<<<<<<< HEAD
         documents = self.get_all_documents(
+=======
+        docs = self._get_all_documents(
+>>>>>>> development
```
Comment (boba-and-beer) [inline relevanceai/vector_tools/cluster.py]: Lol so this is where it broke.

File: relevanceai/dataset_api/dataset.py [hunk 3]
```diff
@@ -795,7 +795,7 @@ def insert_csv(  # type: ignore
         retry_chunk_mult: int
             Multiplier to apply to chunksize if upload fails
         csv_args : dict
-            Optional arguments to use when reading in csv. For more info, see https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html
+            Optional arguments to use when reading in csv. For more info, see https://pandas.pydata.org/documents/reference/api/pandas.read_csv.html
```
Comment (boba-and-beer) [inline relevanceai/dataset_api/dataset.py:798]: This shouldn't have been changed.

File: relevanceai/vector_tools/cluster.py [hunk 4]
```diff
@@ -548,7 +551,7 @@ def kmeans_cluster(
         self.logger.warning(
             "Retrieving documents... This can take a while if the dataset is large."
         )
-        docs = self._get_all_documents(
+        documents = self.get_all_documents(
```
Comment (boba-and-beer) [inline relevanceai/vector_tools/cluster.py:554]: Will need to revert this.

File: relevanceai/vector_tools/cluster_evaluate.py [hunk 5]
```diff
@@ -257,7 +257,7 @@ def _get_cluster_documents(
         else:
             vector_select_field = []
 
-        docs = self._get_all_documents(
+        documents = self.get_all_documents(
```
Comment (boba-and-beer) [inline relevanceai/vector_tools/cluster_evaluate.py:260]: Will need to revert this

File: relevanceai/vector_tools/cluster.py (hunk 1 above)
Comment (charyeezy) [reply relevanceai/vector_tools/cluster.py:689]: Okay will fix ~

**Pull request merged**